cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesBoundary.java

```diff
@@ -7,27 +7,30 @@
 
 package com.powsybl.cgmes.conversion;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
+import com.powsybl.cgmes.conversion.elements.ACLineSegmentConversion;
+import com.powsybl.cgmes.conversion.elements.EquipmentAtBoundaryConversion;
+import com.powsybl.cgmes.conversion.elements.EquivalentBranchConversion;
+import com.powsybl.cgmes.conversion.elements.SwitchConversion;
+import com.powsybl.cgmes.conversion.elements.transformers.TwoWindingsTransformerConversion;
 import com.powsybl.cgmes.model.CgmesModel;
 import com.powsybl.cgmes.model.PowerFlow;
 import com.powsybl.triplestore.api.PropertyBag;
 import com.powsybl.triplestore.api.PropertyBags;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
 
 /**
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  */
+
 public class CgmesBoundary {
 
+    public enum BoundaryEquipmentType {
+        AC_LINE_SEGMENT, SWITCH, TRANSFORMER, EQUIVALENT_BRANCH
+    }
+
```

Comment:
Related to the comment on `BoudaryEquipment`, this enum can be in `BoundaryEquipment`

Reply:
Done